content/en/getting_started/synthetics/browser_test.md

```diff
@@ -39,7 +39,11 @@ The example below demonstrates the creation of a browser test that maps a user's
 
 1. In the Datadog site, hover over **Digital Experience** in the left hand menu and select **[Tests][2]** (under **Synthetic Monitoring & Testing**).
 2. In the top right corner, click **New Test** > **[Browser Test][3]**.
-3. Define your browser test:
+3. You can select a pre-populated template to choose from:
+
+   {{< img src="getting_started/synthetics/synthetics_browser_templates.png" alt="Synthetics browser test landing page with templates" style="width:100%;" >}}
+
+4. OR, click the **+** option to build a test from scratch by performing the following:
```

Comment:
Same structure comment as before! The problem with starting with "or" is that I'm not clear on what it's an alternative to (the previous step? The whole procedure up until this step?).